pkgs/applications/version-management/sourcehut/pages.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchFromSourcehut
+, buildGoModule
+}:
+let
+  version = "0.4.8";
+
+  src = fetchFromSourcehut {
+    owner = "~sircmpwn";
+    repo = "pages.sr.ht";
+    rev = version;
+    sha256 = "sha256-z9w8v5e6LY6VUEczltyD55KEUUH7Gw1vUO00KPmT+D8=";
+  };
+
+in
+buildGoModule {
+  inherit src version;
+  pname = "pagessrht";
```

Comment:
```suggestion
buildGoModule rec {
  pname = "pagessrht";
  version = "0.4.8";

  src = fetchFromSourcehut {
    owner = "~sircmpwn";
    repo = "pages.sr.ht";
    rev = version;
    sha256 = "sha256-z9w8v5e6LY6VUEczltyD55KEUUH7Gw1vUO00KPmT+D8=";
  };

```

Reply:
Done. (with fixing the `rev = version`).